web/components/CommitBlock.jsx

```diff
@@ -18,35 +18,38 @@ class CommitBlock extends React.Component {
     }
 
     upsertSetBlocks = () => {
-        const { selectedDay, unsavedSetBlocks, currentTeamMember } = this.props
+        const { selectedDay, unsavedSetBlocks, currentTeamMember, enableSubmit } = this.props
         const day = moment(selectedDay).format('YYYY-MM-DD');
-        unsavedSetBlocks[day].map( (setBlock) => {
-            if (setBlock.id) {
+        if (enableSubmit) {
+
+            unsavedSetBlocks[day].map( (setBlock) => {
+                if (setBlock.id) {
                 // Update
-                if (setBlock.blockFraction !== 0) {
+                    if (setBlock.blockFraction !== 0) {
                     // Update if change description, issueUrl, or blockFraction
-                    this.props.updateSetBlock(setBlock)
-                } else {
+                        this.props.updateSetBlock(setBlock)
+                    } else {
                     // Delete if blockFraction is 0
-                    this.props.deleteSetblock({ setblockId: setBlock.id })
-                }
-            } else if (setBlock.blockFraction !== 0) {
+                        this.props.deleteSetblock({ setblockId: setBlock.id })
+                    }
+                } else if (setBlock.blockFraction !== 0) {
                 // Create a new one if have blockFraction != 0
-                this.props.createSetBlock({ teamMemberId: currentTeamMember.id, date: day, ...setBlock, issueUrl: (setBlock.issueUrl || '') })
-            }
-        })
+                    this.props.createSetBlock({
+                        teamMemberId: currentTeamMember.id, date: day, ...setBlock, issueUrl: (setBlock.issueUrl || '') 
```

Comment:
For code readability, let's put these properties on separate lines.

General rule of thumb is to use newlines if code starts to go too far off on 1 line. Ideally a developer doesn't have to scroll sideways too much to read code

Reply:
May be we can configure ESLint to solve this, because according to WebStorm (the IDEA that I uses) is the right length according to the good practices, unless ESLint tells it otherwise